src/lightning_app/utilities/app_logs.py

```diff
@@ -92,20 +112,12 @@ def _app_logs_reader(
     for th in log_threads:
         th.start()
 
-    user_log_start = "<<< BEGIN USER_RUN_FLOW SECTION >>>"
-    start_timestamp = None
-
     # Print logs from queue when log event is available
     try:
         while True:
             _, component_name, log_event = read_queue.get(timeout=None if follow else 1.0)
             log_event: _LogEvent
-
-            if user_log_start in log_event.message:
-                start_timestamp = log_event.timestamp + timedelta(seconds=0.5)
-
-            if start_timestamp and log_event.timestamp > start_timestamp:
-                yield component_name, log_event
```

Comment:
Why this is removed ? I think it was added to show the logs after "<<< BEGIN USER_RUN_FLOW SECTION >>>"

Reply:
I added in Adam PR, but it didn't work properly. This would filter only the flow but not the works.

Reply:
I moved it within the callback, so this is filtered for each WebSocket instead.